citesphere/src/main/java/edu/asu/diging/citesphere/core/util/model/impl/CitationHelper.java

```diff
@@ -122,6 +124,22 @@ public void updateCitation(ICitation citation, CitationForm form, IUser user) {
             }
         }
     }
+    
+    /**
+     * Use updateCitationWithCollection(ICitation, String, IUser) method when moving
+     * citation to a collection
```

Comment:
this is not what this method is doing. javadoc should describe what a method is doing and what is important to know when using it, so it can easily be reused.